File: csharp-beginner-workbook/module-07-task-tracker/project/TaskItem.cs
// TaskItem.cs
// This class represents a single task in the Task Tracker application

using System;

class TaskItem
{
    public string Title { get; set; }
    public bool IsCompleted { get; set; }
    
    // Constructor - requires a title when creating a task
    public TaskItem(string title)
    {
        Title = title;
        IsCompleted = false;  // New tasks start as incomplete
    }
    
    // Method to mark task as complete
    public void MarkComplete()
    {
        IsCompleted = true;
    }
    
    // Method to display task information
    public void DisplayInfo(int index)
    {
        string status = IsCompleted ? "✓ Complete" : "○ Incomplete";
        Console.WriteLine($"{index}. {Title} - {status}");
    }
}

File: python-beginner-workbook/module-07-task-tracker/project/task_item.py
# task_item.py
# This class represents a single task in the Task Tracker application

class TaskItem:
    def __init__(self, title):
        self.title = title
        self.is_completed = False  # New tasks start as incomplete
    
    def mark_complete(self):
        """Mark this task as completed."""
        self.is_completed = True
    
    def display_info(self, index):
        """Display task information with index number."""
        status = "✓ Complete" if self.is_completed else "○ Incomplete"
        print(f"{index}. {self.title} - {status}")

